Bio/NMR/xpktools.py

```diff
@@ -313,8 +304,8 @@ def _read_dicts(fn_list, keyatom):
     datalabel_list = []
     for fn in fn_list:
         peaklist = Peaklist(fn)
-        dict = peaklist.residue_dict(keyatom)
-        dict_list.append(dict)
+        dictionary = peaklist.residue_dict(keyatom)
+        dict_list.append(dictionary)
```

Comment:
Why keep the local variable?

```python
for fn in fn_list:
    peaklist = Peaklist(fn)
    dict_list.append(peaklist.residue_dict(keyatom))
```

Could even go shorter, but that begins to lose clarity:

```python
for fn in fn_list:
    dict_list.append(Peaklist(fn).residue_dict(keyatom))
```